shared/js/background/chrome-events.es6.js

```diff
@@ -339,6 +339,19 @@ chrome.webRequest.onBeforeSendHeaders.addListener(
         // Check if origin is safe listed
         const tab = tabManager.get({ tabId: e.tabId })
 
+        // Firefox only - Check if this tab had a surrogate redirect request and if it will
+        // likely be blocked by CORS (Origin header). Chrome surrogate redirects happen in onBeforeRequest.
+        if (browser === 'moz' && tab && tab.surrogates && tab.surrogates[e.url]) {
+            const hasOrigin = e.requestHeaders.filter(h => h.name === 'Origin')
```

Comment:
Not a big deal, but might want to drop case sensitivity here since header names can technically take any case.